pkg/auth/iam/iam.go

```diff
@@ -477,3 +496,29 @@ func respondError(httpStatus, errorCode int, errorMessage string) restful.Servic
 		Message: string(messageByte),
 	}
 }
+
+// SetDevStackTracer used for activate verbose error message in non-prod environment
+// example usage : <package_alias>.SetDevStackTracer(configuration.Realm)
+func SetDevStackTracer(realm string) {
```

Comment:
does that mean all the services need to change their code to call this method to reach the goal, is there any way to omit this step?

Reply:
@arya-ab maybe you can define the DevStack active based on the environment variables chat on the library directly. So later when the services implement the library didn't need to care about setting up this configuration. All they need is to deploy the service using that envar

Reply:
Yup, I just rethink to make it automatically implemented once the service deployed @angelo-chan @rexydwybrundy 
I'll update the code first